src/algorithms/PVT/gnuradio_blocks/rtklib_pvt_gs.cc

```diff
@@ -1813,8 +1823,11 @@ void rtklib_pvt_gs::initialize_and_apply_carrier_phase_offset()
 int rtklib_pvt_gs::work(int noutput_items, gr_vector_const_void_star& input_items,
     gr_vector_void_star& output_items __attribute__((unused)))
 {
+    std::cout << "\nNumber of records in iteration: " << noutput_items;
+    
     for (int32_t epoch = 0; epoch < noutput_items; epoch++)
-        {
+        {   
+            d_total_pvt_measurements++;
```

Comment:
are they still teaching post-increment? That unfortunate habit dates back to the original K&R C book. Pre-increment is better, except in cases where you really need post-increment semantics.